pkg/scheduler/apis/config/types.go

```diff
@@ -186,6 +186,9 @@ type ElasticQuotaArgs struct {
 
 	// EnableCheckParentQuota check parentQuotaGroups' used and runtime Quota in PreFilter
 	EnableCheckParentQuota *bool
+
+	// DisableRuntimeQuota if true, use max instead of runtime for all checks.
+	DisableRuntimeQuota *bool
```

Comment:
Here we can define the field type as Bool instead of bool pointer. The code-gen can generate code converting to bool from pointers. Other fields can be changed together.

Reply:
done